Change `MATLAB Octave` to `MATLAB` to make the languages list look better

Panquesito7 opened the pull request:
Things added/changed:

- Change `MATLAB Octave` to `MATLAB` to make the languages list look better.
  - MATLAB Octave can also be interpreted as `MATLAB`.

![Updated languages list](https://user-images.githubusercontent.com/51391473/120235421-94bb2700-c21f-11eb-8f9c-7c82c25dae9c.png)

Panquesito7 committed:
Change `MATLAB Octave` to `MATLAB`...

...to make the languages list look better.
MATLAB Octave can also be interpreted as `MATLAB`.

raklaptudirm committed:
chore: `AArch64 Assembly` -> `AArch64`

ayaankhan98 commented on lib/repositories.ts:
```diff
@@ -48,7 +48,7 @@ export const Repositories = {
   },
   aarch64_assembly: {
-    name: "AArch64 Assembly",
+    name: "AArch64",
```
can we write `x64 ASM`?

raklaptudirm replied:
@Panquesito7 ?

raklaptudirm committed:
chore: Reorder Languages

Reordered languages for a better view. Rows of 34 (Last 35)

raklaptudirm committed:
chore: Reorder repositories

3 x 34 (+1)

Divisions:
1. 25 6
2. 25 6
3. 25 6
3. 28 5

Breaks:
1. Ruby(4) + PHP(3) + Javascript(10) + C++(3) + C#(2) + Elm(3)
2. AArch64(7) + Rust(4) + Dart(4) + Scala(5) + Java(4) + C(1)
3. Kotlin(6) + OCaml(5) + Swift(5) + MATLAB(6) + Go(2) + R(1)
4. Haskell(7) + Python(6) + F#(2) + Elixir(6) + Jupyter(7)

maltejur commented:
Reordering the repositories directly also leads to a different priority (if there now is a Ruby implementation for a algorithm, it will also be the first to be shown). This also leads to the build error, because on the main page we search for the algorithm `quick-sort`, but it is now called `quicksort`, because that is the name that gets resolved from the ruby filename.

If you want to change the order, you would probably have to do it here:
https://github.com/TheAlgorithms/website/blob/59ea172afaf12ff734b816fcb5439e33cf1df50a/pages/index.tsx#L131

But maybe we should rework the whole languages list completely.

raklaptudirm commented:
@maltejur Changing the order in `index.tsx` would render `repositories.ts` useless. We do need to rework the languages list. Should I switch back to the working commit or will we do the rework in this `PR`? Also, what's your view on renaming `AArch 64 Assembly` to `x64 Asm`?

maltejur commented:
> @maltejur Changing the order in `index.tsx` would render `repositories.ts` useless.

The file is also used in many other places, thats why this change broke some other things, but that doesn't really matter.

> Should I switch back to the working commit or will we do the rework in this `PR`? 

I have already worked on a new list yesterday and pushed the branch now here:
https://github.com/TheAlgorithms/website/tree/improve_languages_list
I will open a new PR for that, so we can close this one. Also, what do you think about the new list? (you can take a look [here](https://website-4jx1jwkzi-thealgorithms.vercel.app/)) I am not sure if it is the right approach, so I would love to hear what you think about it.

> Also, what's your view on renaming `AArch 64 Assembly` to `x64 Asm`?

There is a difference between x64 Assembly and AArch64. With the new approach, it also shouldn't matter that the text is longer.
